fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -14,13 +14,23 @@ def self.run(params)
         end
         UI.user_error!("Please provide an api_token using api_token:") unless params[:api_token]
 
-        command = []
-        command << File.expand_path(params[:binary_path])
-        command << "-a #{params[:api_token]}"
-        command << "-p #{params[:platform]}"
-        command << File.expand_path(params[:dsym_path])
+        dsym_paths = []
+        dsym_paths << params[:dsym_path]
+        dsym_paths += Actions.lane_context[SharedValues::DSYM_PATHS] if Actions.lane_context[SharedValues::DSYM_PATHS]
 
-        return Actions.sh command.join(" ")
+        UI.error("Couldn't find any dSYMs, please pass them using the dsym_path option") if dsym_paths.count == 0
+
+        results = []
+        dsym_paths.each do |current_path|
+          command = []
+          command << File.expand_path(params[:binary_path])
+          command << "-a #{params[:api_token]}"
+          command << "-p #{params[:platform]}"
+          command << File.expand_path(current_path)
+          results << Actions.sh(command.join(" "))
+        end
+
+        return results
```

Comment:
out of curiosity, what are the conventions for return values in actions ? Given that now we go from value to []